k9mail-library/src/main/java/com/fsck/k9/mail/store/imap/FolderNotFoundException.kt

```diff
@@ -0,0 +1,6 @@
+package com.fsck.k9.mail.store.imap
+
+import com.fsck.k9.mail.MessagingException
+
+class FolderNotFoundException(val folderServerId: String)
```

Comment:
we will end up with a lot of files with two relevant lines if we arrange them like this. it would however be nice to have more specialized MessagingException subclasses. Maybe we can group those, e.g. in this case into an ImapExceptions file?